pkg/auth/rbac.go

```diff
@@ -224,3 +224,11 @@ func CheckUserPermissions(rbacConfig RBACConfig, user *User, env, envGroup, appl
 	p := fmt.Sprintf(PermissionTemplate, user.DexAuthContext.Role, action, envGroup, env, application)
 	return status.Errorf(codes.PermissionDenied, fmt.Sprintf("user does not have permissions for: %s", p))
 }
+
+// Helper function to parse the scopes.
+func ReadScopes(s string) (scopes []string) {
+	// Removes the array brackets.
+	replacer := strings.NewReplacer("[", "", "]", "", " ", "")
```

Comment:
if the brackets are such a hassle, can we just remove them? i.e. just have a comma separated list?
Then you can just do `strings.Split`.

Reply:
Yes, I can do that. I added a comment for it as well